docs/sources/developers/plugins/migration-guide.md

```diff
@@ -8,7 +8,7 @@ title: Plugin migration guide
 
 ## Introduction
 
-This guide helps you identify the steps you need to take based on the Grafana version your plugin supports and explains how to migrate the plugin to the 8.2.x or a later version.
+This guide helps you identify the steps you need to take based on the Grafana version your plugin supports and explains how to migrate the plugin to the 9.x or a later version.
```

Comment:
Not sure we should need to keep changing this intro. Maybe we could use something more generic?

```suggestion
The guide helps you identify the steps required to update a plugin from the Grafana version it currently supports to newer versions of Grafana.
```